### DISCUSSION TITLE
[Help]: Autoscroll carousel slight jittering in firefox

### DISCUSSION THREAD

**Discussion by karlodr** [davidjerleke/embla-carousel#1066] [Help] [answered]
### Summary

Hello, I have been trying to resolve this jittering issue, but it seems I can't remove it easily.

The best I have managed is to reduce the speed from 1 to 0.5. For some images, it is still very noticeable.

It only happens in Firefox, but it works perfectly fine in Chrome.

Please check this video for reference:
https://drive.google.com/file/d/1kQdAIIoVp86pdYwJ_B6b3BO93RSNPZg7/view?usp=sharing

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [X] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
@karlodr could you also share your setup in a CodeSandbox? Thanks.

**Reply by karlodr** [reply]
Hello, we tried upgrading the version we are using from 8.3 to 8.5, and it seems to fix the issue. However, there is still some slight jitter in firefox, but it happens very rarely. You can check this sandbox; it replicates the same structure we have in our code:

Sandbox:
https://ctmmnp.csb.app/

**Reply by davidjerleke** [reply]
@karlodr have you tried setting this on the container?

```css
.embla__container {
  will-change: transform;
}
```

**Reply by karlodr** [reply]
Yes, that style is already present in our code. I updated my sandbox and applied it to the `.embla__container`.

**Reply by davidjerleke** [reply]
@karlodr I see. Please try [this sandbox](https://hft7d8.csb.app/) and let me know if there's any improvement. Please note that the resize functionality is broken in the same example though so don't resize the window. This is just to test things. Thanks.

**Reply by karlodr** [reply]
Thanks @davidjerleke, the code update improved things. It seems like the issue might be on our side now. So far, all my concerns have been addressed, and I don't need anything further at the moment. Thank you so much for your help!

**Comment by karlodr** [comment, marked as answer]
Just summarizing the steps we followed to resolve the issue:
- Upgraded the package version from `8.3` to `8.5`.
- Modified the code to set `playOnInit` to `true` after the component has successfully loaded.

Thanks to @davidjerleke for the help.
